src/main/resources/public/partials/graphs.html

```diff
@@ -64,7 +64,7 @@
 </div>
 <hr>
 <div class="row">
-  <div class="col-sm-12">
+  <div class="col-xs-8 col-xs-offset-2 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
```

Comment:
In chart.js version > 2.4.0 the behaviour of the size has changed, so I decided to set a size.